mkdocs.yml

```diff
@@ -110,6 +110,8 @@ nav:
     - Legacy:
       - './design/legacy-design/README.md'
       - './design/legacy-requirements/README.md'
+  - Definitions:
```

Comment:
I'm struggling with this being its own tab on the site -- would you be open to having this as a category under introduction? It seems like it would fit there and there doesn't seem to be enough content to warrant its own tab at the top.